### PR TITLE
Enable PDF report generation on Mac OS

### PR DESCRIPTION
**Pull request by abhargav-mw**
This PR is for enabling PDF Report generation on Mac OS Platform.

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Argument                  | Description
...
 `selectByFolder`          | (Optional) Location of the folder used to select test suite elements, relative to the project root folder. To create a test suite, MATLAB uses only the tests in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or semicolon-separated list.<br/>**Example:** `test`<br/>**Example:** `test/folderA; test/folderB`
 `selectByTag`             | (Optional) Test tag used to select test suite elements. To create a test suite, MATLAB uses only the test elements with the specified tag.<br/>**Example:** `Unit`
-`testResultsPDF`          | (Optional) Path to write test results report in PDF format. Currently, this argument is not supported on macOS platforms.<br/>**Example:** `test-results/results.pdf`         
+`testResultsPDF`          | (Optional) Path to write test results report in PDF format. On macOS platforms, this input is supported in MATLAB R2020b and later.<br/>**Example:** `test-results/results.pdf`
```
[the remark this answers is not in the document]
Comment (mw-hrastega) [reply overview.md:117]: It's common in MathWorks doc to include "platform" or "system" after the name of the operating system.

Comment (mw-hrastega) [inline overview.md:117]: input -> argument

Is there a reason to use input? The table header mentions "argument."

File: tasks/run-matlab-tests/v0/task.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
             "defaultValue": "",
             "groupName": "testArtifacts",
             "required": false,
-            "helpMarkDown": "Path to write test results report in PDF format (currently not supported on macOS platforms)."
+            "helpMarkDown": "Path to write test results report in PDF format (on macOS platforms this is supported in MATLAB R2020b and later)."
```
Comment (mw-hrastega) [inline tasks/run-matlab-tests/v0/task.json:64]: (requires MATLAB R2020b or later on macOS platforms).

**Comment by mw-hrastega** [comment]
Could you also remove "> :information_source: **Note:** ", which appears immediately after the table? The icon does not render correctly in Azure MarketPlace:

https://marketplace.visualstudio.com/items?itemName=MathWorks.matlab-azure-devops-extension

File: overview.md (hunk 1 above)
Comment (abhargav-mw) [reply overview.md]: Done.

File: tasks/run-matlab-tests/v0/task.json (hunk 2 above)
Comment (abhargav-mw) [reply tasks/run-matlab-tests/v0/task.json]: Done

**Review by mcafaro — APPROVED** [review]
Ship it!

**Review by mw-hrastega — APPROVED** [review]

**Pull request merged**